tests/functional/test_ams_inference.py

```diff
@@ -157,6 +171,122 @@ def test_ams_with_wrong_model_name(self, start_ams_service_with_wrong_model_name
+        highest_value = ""
+        type_name = ""
+
+        for classification in response_json["classifications"][0]["attributes"]:
+            if classification["confidence"] > highest_probability:
+                highest_probability = classification["confidence"]
+                highest_value = classification["value"]
+                type_name = classification["name"]
+
+        assert highest_probability > 0.547
+        assert highest_value == "red"
+        assert type_name == "color"
+
+        highest_probability = 0.0
+        highest_value = ""
+        type_name = ""
+
+        for classification in response_json["classifications"][1]["attributes"]:
+            if classification["confidence"] > highest_probability:
+                highest_probability = classification["confidence"]
+                highest_value = classification["value"]
+                type_name = classification["name"]
+
+        assert highest_probability > 0.9421
+        assert highest_value == "truck"
+        assert type_name == "type"
+
+
+    def test_vehicleDetection(self, start_ams_service, object_detection_image_two_entities):
+        with open(object_detection_image_two_entities, mode='rb') as image_file:
+            image_bytes = image_file.read()
+        _, ports = start_ams_service
+        ams_port = ports['port']
+
+        endpoint_url = "http://localhost:{}/{}".format(ams_port, "vehicleDetection")
+        response = requests.post(endpoint_url,
+                                headers={'Content-Type': 'image/png',
+                                        'Content-Length': str(len(object_detection_image_two_entities))},
+                                data=image_bytes)
+        assert response.status_code == 200
+        assert response.headers.get('Content-Type') == 'application/json'
+
+        response_json = response.json()
+
+        highest_probability = 0.0
+        highest_box = dict()
+        tag_value = ""
+        detections_count = 0
+
+        for detection in response_json["entities"]:
+            detections_count += 1
+
+            if detection["tag"]["confidence"] > highest_probability:
+                highest_probability = detection["tag"]["confidence"]
+                highest_box = detection["box"]
+                tag_value = detection["tag"]["value"]
+
+        epsilon = 0.00000000001
+        assert highest_probability > 0.67
+        assert tag_value == "vehicle"
+        assert detections_count == 11
+        assert highest_box["w"] + epsilon >= 0.034460186958313
+        assert highest_box["l"] + epsilon >= 0.783527314662933
+        assert highest_box["h"] + epsilon >= 0.0431380569934845
+        assert highest_box["t"] + epsilon >= 0.173053205013275
+        assert highest_box["w"] - epsilon <= 0.034460186958313
+        assert highest_box["l"] - epsilon <= 0.783527314662933
+        assert highest_box["h"] - epsilon <= 0.0431380569934845
+        assert highest_box["t"] - epsilon <= 0.173053205013275
```

Comment:
I think it would be more clear to assert expected range in one statement, e.g.:
```python
assert 0.034460186958313 - epsilon <= highest_box["w"]  <= 0.034460186958313 + epsilon
```
or:
```python
assert abs(highest_box["w"] - 0.034460186958313) <= epsilon 
```

I think we should not test to have such precision in bounding box coordinates - it may differ due to rounding errors on different hardware, so perhaps we should have more tolerance for the results in tests.